.github/workflows/nodejs.yml

```diff
@@ -47,3 +47,21 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: yarn
     - run: yarn build:docs
+
+  build-npm-release:
```

Comment:
I think this job merits a comment explaining why it's here.

Reply:
Updated!